packages/scripts/utils/env.js

```diff
@@ -4,6 +4,17 @@ const fs = require('fs');
 const get = require('lodash.get');
 const path = require('path');
 
+/**
+ * Get talend-scripts.json
+ */
+function getScriptsConfig() {
+	const userConfigFilePath = path.join(process.cwd(), 'talend-scripts.json');
+	if (fs.existsSync(userConfigFilePath)) {
+		return require(userConfigFilePath);
```

Comment:
Be careful with several `require` executions, require has a cache system.
https://stackoverflow.com/questions/9210542/node-js-require-cache-possible-to-invalidate

Reply:
The goal is not to support any change on the fly. Does the cache persist between calls ?
 What I understand is that you run your process, and everytime to require, it returns the one cached. It is cached the first time it is called --> within the process.

Reply:
Right